spec/mailers/notification_mailer_spec.rb

```diff
@@ -3,56 +3,56 @@
 
 describe NotificationMailer do
   describe '#daily_summary' do
-    let(:user) { FactoryGirl.create(:user) }
+    let(:user) { FactoryGirl.build(:user) }
```

Comment:
Similar comment to https://github.com/mysociety/alaveteli/pull/4572/commits/154f58bc1d6c293cf6c060b0fe1a4992688925d9 – _why_ can we use build instead of create? I think its the same situation where these built records get saved elsewhere?